.github/workflows/update-helm-chart-faas-images.yaml

```diff
@@ -0,0 +1,66 @@
+name: Update FaaS Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      node_v4:
+        description: 'Update Node v4 (true/false)'
+        required: true
+        default: 'true'
+      node_v5:
+        description: 'Update Node v5 (true/false)'
+        required: true
+        default: 'false'
+      rego_v5:
+        description: 'Update Rego v5 (true/false)'
+        required: true
+        default: 'true'
+      tag:
+        description: 'Image Tag'
+        required: true
+        default: 'testing'
+
+jobs:
+  update-faas-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.CLOUDENTITY_CI_TOKEN }}
+      - name: Set Environment Variables
+        run: |
+          echo "NODE_V4=${{ github.event.inputs.node_v4 }}" >> $GITHUB_ENV
```

Comment:
Did you consider using a matrix (to have an execution per environment)?

Reply:
No, the parameters are dynamic passed through input variables and all my attempts to make a loop based on inputs directly have failed. I don't want to spend any more time on it tbh